### ISSUE TITLE
Provide Coverage

### ISSUE THREAD

**Issue by erayerdin** [abhinavk99/jikanpy#19]
This is a post-step of Travis. Community might want to see the percentage of covered line via tests.

[Codecov](https://codecov.io) is an awesome tool about this. You might want to check out.

This issue also can wait a little longer.

**Comment by abhinavk99** [comment]
Would you be willing to make a PR for this as well? Finals are coming up so I'm kind of busy.

**Comment by erayerdin** [comment]
Of course, I will provide coverage. I was just waiting for #20 to be merged, which was an important step for coverage.

**Comment by erayerdin** [comment]
Okay, so I've checked if I have done some mistake, but turns out I didn't.

I have found out that Travis itself (not Shields) shows the project as unknown. I have come across [this question in SO](https://stackoverflow.com/questions/12030304/why-is-my-travis-ci-build-status-always-reflected-as-unknown) and tried some solutions (like explicitly defining master, but it didn't succeed).

[This answer](https://stackoverflow.com/a/48632145) particularly suggests that @AWConant might have to trigger a manual build. It is under "More Options" section.

I also thought about cache problems but it has been 10 hours since the build passed, so that is a stupid idea. If it is, all we have to do is to wait. (?)

Also, today, Github has been having a problem, Travis has also an orange icon at the bottom, [this](https://www.traviscistatus.com) is the current status of what's going on between Github and Travis (Travis Status also has [a Twitter account](https://twitter.com/traviscistatus).). Maybe, that caused a problem. (The second issue on Github this month, how annoying.)

All in all, the issue must be resolved (i) after Github resolves its problems or (ii) after the next PR or commit on repository. If @AWConant wants to trigger a new build by hand, he needs to wait for Github to solve its issues, we can guess at least 24 hours later of this comment, it will be safer.
